src/components/book-content-mixin.cjsx

```diff
@@ -63,7 +63,7 @@ module.exports =
     )
 
   isMediaLink: (link) ->
-    link.hash.length > 0 and link.hash.search('/') is -1
+    (link.hash.length > 0 and link.hash.search('/') is -1) or link.href isnt link.getAttribute('href')
 
```

Comment:
:question: Huh?  I'm confused by how `link.href` would ever differ from `link.getAttribute('href')`  I'm sure it must though since you added the check deliberately.